Sources/SQLiteKit/SQLiteConnectionSource.swift

```diff
@@ -48,3 +44,31 @@ public struct SQLiteConnectionSource: ConnectionPoolSource {
 }
 
 extension SQLiteConnection: ConnectionPoolItem { }
+
+fileprivate extension String {
+    var asSafeFilename: String {
+        #if os(Windows)
+        self.replacingOccurrences(of: ":", with: "_").replacingOccurrences(of: "\\", with: "-")
+        #else
+        self.replacingOccurrences(of: "/", with: "-")
+        #endif
+    }
+}
```

Comment:
👌

Reply:
I even consulted with Saleem to end up with this 😉